wandb/cli/cli.py

```diff
@@ -2086,8 +2094,15 @@ def put(path, name, description, type, alias):
     else:
         raise ClickException("Path argument must be a file or directory")
 
+    if run_id is not None:
```

Comment:
It looks like if you add a `run_id`, then we ignore `resume=False` and resume anyway?

1. If you don't include a run_id, then resume=True would be an error, and resume=False can be inferred
2. If you do include a run_id, then it would matter; but maybe the default should be True, and if it's False and there's a run_id we know there's an explicit intent to create a new run.

I think we can also see if a run with that id exists (if we want to) and resume if it does or create a new one if it doesn't; I think anything else would be an error.

Reply:
So you're saying its probably better to specify both the `run_id` and `resume=True` to resume a run with that run_id. 

Ahh you're right. I misunderstood the current behavior when I referred to the [documentation](https://docs.wandb.ai/ref/python/init#docusaurus_skipToContent_fallback) for `wandb.init()`. `resume` is `None` by default.

Reply:
So I guess the `resume` option shouldn't be a flag either, as it's not boolean?